middleware/limiter/limited_fixed.go

```diff
@@ -79,7 +76,7 @@ func (FixedWindow) New(cfg Config) fiber.Handler {
 		if remaining < 0 {
 			// Return response with Retry-After header
 			// https://tools.ietf.org/html/rfc6584
-			c.Set(fiber.HeaderRetryAfter, strconv.FormatUint(expire, 10))
+			c.Set(fiber.HeaderRetryAfter, strconv.FormatUint(expiration, 10))
```

Comment:
think this is wrong, here should be the remaining time and not the complete time 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After

Reply:
It is wrong indeed. 122cbfeeaca88dd8abf78fe7702d7a978bdf09bf is the only commit now, it holds the proposed doc fix for both window limiters.